python/cudf/cudf/tests/test_string.py

```diff
@@ -36,6 +36,7 @@
 idx_list = [None, [10, 11, 12, 13, 14]]
 
 idx_id_list = ["None_index", "Set_index"]
+rng = np.random.default_rng(seed=0)
```

Comment:
*nit*: avoid global rng object, although I see (and I guess this occurs throughout) this is for generating values for the parametrisations, so it can't obviously live inside a test.

Reply:
Avoiding using globals now.